### PR TITLE
lib/ffmpeg/transcoderbase.py add HDR test

### PR DESCRIPTION
**Pull request by jian2x**
Signed-off-by: ZhouJian <jian2x.zhou@intel.com>

### CODE REVIEW

**Comment by jian2x** [comment]
> about ffmpeg format to consider

Thanks, I'll refer to it and make changes

File: lib/ffmpeg/transcoderbase.py [hunk 1]
```diff
@@ -210,6 +211,18 @@ def transcode(self):
           iopts.format(osencoded), oopts.format(vppscale, self.frames, osyuv))
         self.check_resolution(output, osencoded)
         self.check_metrics(yuv, refctx = [(n, channel)])
+
+        is_hdr = vars(self).get("HDR", 0)
```
Comment (uartie) [inline lib/ffmpeg/transcoderbase.py:215]: Should there be a per-output option?  Maybe some case can specify a codec that is unable to transcode HDR meta.

Comment (uartie) [reply lib/ffmpeg/transcoderbase.py:215]: also, please use lowercase for variable name: `HDR` -> `hdr`

File: lib/ffmpeg/transcoderbase.py [hunk 2]
```diff
@@ -210,6 +211,18 @@ def transcode(self):
...
+        if is_hdr:
+          output = call(
+            f"{exe2os('ffprobe')} -i {osencoded}"
+            f" -show_streams"
+          )
+          color_range_info = re.findall(r'(?<=color_range=)\w*', output)
+          color_space_info = re.findall(r'(?<=color_space=)\w*', output)
+          color_transfer_info = re.findall(r'(?<=color_transfer=)\w*', output)
+          color_primaries_info = re.findall(r'(?<=color_primaries=)\w*', output)
```
Comment (uartie) [inline lib/ffmpeg/transcoderbase.py:224]: Does this HDR meta change when it is transcoded?  If not, we should verify it matches the source HDR meta.

**Comment by jian2x** [comment]
@uartie According to your suggestion, I made some changes, Please help me review it again

**Pull request merged**